app/services/mapping/governmentagencygoodsitem/PackagingBuilder.scala

```diff
@@ -25,40 +28,70 @@ import wco.datamodel.wco.declaration_ds.dms._2.{
 
 import scala.collection.JavaConverters._
 object PackagingBuilder {
```

Comment:
Similar to above, can we make this a class and inject the dependent builders

Reply:
this is TODO as well as the PR title suggested this is PART1